client/verta/tests/unit_tests/registry/test_check_model_dependencies.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+""" Unit tests for verta.registry.check_model_dependencies """
+
+import pytest
+from typing import Set
+import warnings
+
+from verta.registry.check_model_dependencies import check_model_dependencies
```

Comment:
This doesn't seem to match the design doc, nor the example in `check_model_dependencies`'s docstring

> ```python
> from verta.registry import check_model_dependencies
> ```